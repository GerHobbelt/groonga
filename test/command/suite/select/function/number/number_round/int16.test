plugin_register functions/number

table_create Data TABLE_NO_KEY
column_create Data value COLUMN_SCALAR Int16

load --table Data
[
{"value": 11},
{"value": -11}
]

select Data --output_columns 'number_round(value)'
select Data --output_columns 'number_round(value, -1)'
