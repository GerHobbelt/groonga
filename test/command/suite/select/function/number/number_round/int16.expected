plugin_register functions/number
[[0,0.0,0.0],true]
table_create Data TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Data value COLUMN_SCALAR Int16
[[0,0.0,0.0],true]
load --table Data
[
{"value": 11},
{"value": -11}
]
[[0,0.0,0.0],2]
select Data --output_columns 'number_round(value)'
[[0,0.0,0.0],[[[2],[["number_round",null]],[11],[-11]]]]
select Data --output_columns 'number_round(value, -1)'
[[0,0.0,0.0],[[[2],[["number_round",null]],[10],[-10]]]]
